### DISCUSSION TITLE
Добыча Темной материи

### DISCUSSION THREAD

**Discussion by dendyua** [Yaro2709/New-Star#54] [General]
Темная материя не добывается при любых условиях, комбинациях флота, уровня экспедиции, состава флота, протяженности экспедиции и т.д.

**Comment by Yaro2709** [comment]
Напишите на своем родном языке. Я не понимаю о чем вы. Спасибо :)

**Reply by dendyua** [reply]
> Напишите на своем родном языке. Я не понимаю о чем вы. Спасибо :)

Это мой родной язык)
Провел около 40 экспедиций, и в Туманность также, и не смог добыть **ни одной единицы Темной материи**. 
Флот был от 100 Больших Транспортников до 9999 штук, а также учувствовали и боевые корабли при полете в Туманность.
